### DISCUSSION TITLE
APIクライアントの作成のIPアドレスについて

### DISCUSSION THREAD

**Discussion by kozake** [ws-4020/mobile-app-crib-notes#292] [Show and tell]
IPは環境変数となるので、環境ごとに変更する方法が提示されれば嬉しいと感じました。

**Comment by tani-takanori** [comment]
Example Appに持っていく話だと思うのでDiscussionに持っていく

**Comment by creature-water-valley** [comment]
環境ごとの設定値に関しては、以下のページに記載済み（Expo configの[extra](https://docs.expo.dev/versions/latest/config/app/#extra)属性の利用）
- [アプリ特有の設定値](https://ws-4020.github.io/mobile-app-crib-notes/react-native/santoku/development/build-configuration/app-constants)
